app/main/__init__.py

```diff
@@ -13,6 +16,12 @@
 migrate = Migrate()
 
 def create_app(config_name):
+  sentry_sdk.init(
+    dsn=os.getenv('sentry_sdk_dsn'),
```

Comment:
will 'sentry_sdk_dsn' be set in the localdev environment? if it is 'none' will the app still start?

Reply:
Yes that's the idea - I've already added the parameters into SSM:

```
aws ssm put-parameter --region $REGION --name "/qa/alegre/sentry_sdk_dsn" --type String --value "https://SOMETHING@OTHER.ingest.sentry.io/THING"
aws ssm put-parameter --region $REGION --name "/live/alegre/sentry_sdk_dsn" --type String --value "https://SOMETHING@OTHER.ingest.sentry.io/THING"
```

The DSNs will be identical for qa and live - the environments will denote the separate environments (i.e. we. aren't going to have a prod project as separate from a dev project in sentry). I believe that, when `dsn` is `None`, it will just start up but not log to sentry, but am in the middle of confirming right now.

Other thing to note is we'll do this in a two-phase deploy - first, deploy to write to both, then remove errbit once we confirm we're good to go.